service-monitor/src/main/java/com/yahoo/vespa/service/slobrok/SlobrokMonitorManagerImpl.java

```diff
@@ -139,6 +135,7 @@ private boolean wouldMonitor(ApplicationId applicationId) {
             case "qrserver":
             case "container":
             case "container-clustercontroller":
+            case "logserver-container":
```

Comment:
How many logserver-containers services/processes/configids are there in an application? Always One?

Reply:
0 or 1 (only Vespa 6 applications have 0)